pkg/controller/servicebindingrequest/service.go

```diff
@@ -18,12 +18,12 @@ import (
 )
 
 var (
-	// ErrUnspecifiedBackingServiceNamespace is returned when the namespace of a service is
+	// errUnspecifiedBackingServiceNamespace is returned when the namespace of a service is
 	// unspecified.
-	ErrUnspecifiedBackingServiceNamespace = errors.New("backing service namespace is unspecified")
+	errUnspecifiedBackingServiceNamespace = errors.New("backing service namespace is unspecified")
 	// EmptyBackingServiceSelectorsErr is returned when no backing service selectors have been
```

Comment:
errEmptyBackingServiceSelectors

Reply:
Solved in 5421402409c79d1dc685c4543a4f378d4a1a04bd.